src/lib/tabs/tabHome/HomeTab.tsx

```diff
@@ -141,13 +149,18 @@ function CardImage(props: { id: string }) {
           outline: 'rgba(255, 255, 255, 0.15) solid 1px',
           boxShadow: 'rgb(0 0 0 / 50%) 2px 2px 3px',
         }}
-        src={Assets.getHomeFeature(props.id)}
+        src={Assets.getHomeFeature(props.id, i18next.resolvedLanguage)}
+        onError={(e) => {
+          e.target.src = Assets.getHomeFeature(props.id)
+          e.onerror = null // prevent infinite looping if for some reason the english image can't be loaded
+        }}
```

Comment:
Is this something you ran into? What was causing the loop?

Reply:
not something I actively ran into, but my understanding is that in theory if the user were able to load the page but not the fallback assets (the english ones) then the onError would run repeatedly since it would never successfully fetch an image

Reply:
Got it, if we're using translated images in multiple files down the line we should consider making an <I18nImage> component or something to wrap this or at least extract the error handler out.